Tests/SentryTests/TestClient.swift

```diff
@@ -82,6 +82,7 @@ class TestClient: Client {
     var captureCrashEventInvocations = Invocations<(event: Event, scope: Scope)>()
     override func captureCrash(_ event: Event, with scope: Scope) -> SentryId {
         captureCrashEventInvocations.record((event, scope))
+        print("### Captured")
```

Comment:
`l`: Did you maybe forget to remove that after debugging?
```suggestion
```

Reply:
For sure.